Index.html

```diff
@@ -27,5 +27,113 @@ <h4 class="connect">LET'S CONNECT</h4>
       <li><img src="Assests/Vectorangerllist.svg"></li>
     </ul>
   </section>
+  <section class="worksSection">
+    <div class="space"></div>
+    <div class="firstCard">
+      <div class="snapshot"></div>
+      <h3 class="tonic">Tonic</h3>
+      <ul class="canopyList">
+        <li class="canopy">CANOPY</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="backEnd">Back End Dev</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="works">
+        <button class="htmlButton buttons"></button>
+        <button class="cssButton buttons"></button>
+        <button class="jsButton buttons"></button>
+      </div>
+      <button class="seeProject">See project</button>
+    </div>
+    <div class="space"></div>
+    <div class="space"></div>
+    <div class="firstCard">
+      <div class="snapshotTwo"></div>
+      <h3 class="stories">Multi-Post Stories</h3>
+      <ul class="canopyList">
+        <li class="canopy">CANOPY</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="backEnd">Back End Dev</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="works">
+        <button class="htmlButton buttons"></button>
+        <button class="cssButton buttons"></button>
+        <button class="jsButton buttons"></button>
+      </div>
+      <button class="seeProject">See project</button>
+    </div>
+    <div class="space"></div>
+    <div class="space"></div>
+    <div class="firstCard">
+      <div class="snapshotThree"></div>
+      <h3 class="tonic">Tonic</h3>
+      <ul class="canopyList">
+        <li class="canopy">CANOPY</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="backEnd">Back End Dev</li>
+        <li class="counter"><img src="Assests/Countercounter.svg"></img></li>
+        <li class="year">2015</li>
+      </ul>
+      <p class="description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <div class="works">
+        <button class="htmlButton buttons"></button>
+        <button class="cssButton buttons"></button>
+        <button class="jsButton buttons"></button>
+      </div>
+      <button class="seeProject">See project</button>
```

Comment:
[OPTIONAL]
- [ ] please make sure your button has the exact shape and size like the figma template below is how the button should look and how your button currently looks. the correct specifications for this button can be found [here](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=Mr0WKx6bVRIlwjv8-0) kindly look at them and add them to your css.
- figma design
![f-b-1](https://user-images.githubusercontent.com/89055502/235336523-20eb68c7-8161-4b8d-bafa-6705cf608a76.png)
- your design
![s-b-1](https://user-images.githubusercontent.com/89055502/235336530-72e99ca9-8707-41cc-86cb-59343ef9bfbe.png)